src/Modal.js

```diff
@@ -235,6 +236,8 @@ class Modal extends React.Component {
   }
 
   init() {
+    const mountNode = this.props.mountNode || document.body;
```

Comment:
could probably use `defaultProps` to default it to `document.body`

Reply:
Look at the my first commit. I used `defaultProps` but when I run `npm start` it threw an error where `document` was `undefined`.

Reply:
Probably webpack-dev-server works in kind of SSR mode?

Reply:
Yeah, I suppose that will happen on the server when document is not defined. What you have here should be fine then.